F1/F1/Login/LoginViewController.swift

```diff
@@ -9,32 +9,27 @@ import UIKit
 
 class LoginViewController: UIViewController {
 
+    // MARK: IBOutlets
     @IBOutlet weak var username: UITextField!
     @IBOutlet weak var password: UITextField!
-    @IBOutlet weak var errorMsg: UILabel!
+    @IBOutlet weak var errorMessage: UILabel!
+
+    // MARK: Variables
     private lazy var viewModel = LoginViewModel()
 
+    // MARK: Function
     override func viewDidLoad() {
-        errorMsg.isHidden = true
+        errorMessage.isHidden = true
         super.viewDidLoad()
     }
 
+    // MARK: IBAction
     @IBAction func loginAction(_ sender: Any) {
-        if viewModel.validDetails(givenUsername: username.text ?? "", givenPassword: password.text ?? "") {
-            errorMsg.isHidden = true
-            performSegue(withIdentifier: "showTabSegue", sender: self)
+        if viewModel.validDetails(givenUsername: username.text, givenPassword: password.text) {
+            errorMessage.isHidden = true
+            performSegue(withIdentifier: Identifiers.showTabSegue, sender: self)
```

Comment:
If you want to clean it up a bit more, this perform segue can be done in the viewModel using a delegate

Reply:
Added a delegate so no need to remove now 😄

Reply:
Can you just check I did that right? 😄